pkg/commands/requests/actions/new.go

```diff
@@ -23,7 +23,7 @@ const (
 	resourceFlagName             = "resources"
 	permissionFlagName           = "permissions"
 	justificationFlagName        = "justification"
-	noInteractiveFlagName        = "no-interactive"
+	interactiveFlagName          = "interactive"
```

Comment:
I feel it's not good that some commands use the `"no-interactive"` flag and others `"interactive"`
I think they better be aligned in names (can be different in values)

Reply:
I removed the `no-interactive` flag from the other command the second PR